autoload/wispath.vim

```diff
@@ -70,11 +70,15 @@ enddef
 
 export def GetCompletion(target_path: string, cursor_col: number, in_cmdline: bool): list<any>
   var completions: list<string>
-  if IsInCmdwin()
-    completions = EvalInAlterWin(function('getcompletion', [target_path, 'file']))
-  else
-    completions = getcompletion(target_path, 'file')
-  endif
+  try
+    if IsInCmdwin()
+      completions = EvalInAlterWin(function('getcompletion', [target_path, 'file']))
+    else
+      completions = getcompletion(target_path, 'file')
+    endif
+  catch
```

Comment:
Please handle only E220 error here and let the other error be reported by introducing a new `:catch` block.  https://vim-jp.org/vimdoc-ja/eval.html#:catch

Reply:
Thank you.
I've handle E220 and E219 exception.
https://github.com/mityu/vim-wispath/pull/5/commits/4533aae5e6347c831ad0f4fabf4c5353332aac15

Could you take another look?